python/spacetime/managers/version_graph.py

```diff
@@ -154,4 +168,75 @@ def maintain(self, state_to_ref, merger_function):
         # First divergent.
         self.maintain_nodes(state_to_ref, merger_function, False)
         # The master line.
-        self.maintain_nodes(state_to_ref, merger_function, True)
\ No newline at end of file
+        self.maintain_nodes(state_to_ref, merger_function, True)
+
+
+class VersionGraphProcess(Process):
+
+    def __init__(self):
+        self.graph = Graph()
+        super().__init__()
+        self.manager = Manager()
+        self.result_manager = Manager()
+        self.event_manager = Manager()
+        self.read_write_queue = self.manager.Queue()
+        self.request_list = []
+
+        def create_flask_app(self):
+            app = Flask(__name__)
+            #app.config["DEBUG"] = True
+
+            @app.route('/next', methods=['GET'])
+            def graph():
+                req = self.read_write_queue.get()
+                self.request_list.append(req[0])
+                if req[0] == "continue_chain":
+                    from_version, to_version, package, e1 = req[1:]
+                    self.process_continue_chain(from_version, to_version, package, e1)
+                elif req[0] == "maintain":
+                    state_to_ref, merger_function, e2 = req[1:]
+                    self.process_maintain(state_to_ref, merger_function, e2)
+                elif req[0] == "get_item":
+                    key, result_queue = req[1], req[2]
+                    self.process_get_item(key, result_queue)
+                return self.graph.convert_to_json()
+
+            @app.route('/displayQueue', methods=['GET'])
+            def display_request_queue():
+                request_string = str(self.request_list)
+                return request_string
+
+            return app
+        self.app = create_flask_app(self)
+        self.start()
+
+    def process_continue_chain(self, from_version, to_version, package, e1):
+        self.graph.continue_chain(from_version, to_version, package)
+        e1.set()
+
+    def process_maintain(self, state_to_ref, merger_function, e2):
+        self.graph.maintain(state_to_ref, merger_function)
+        e2.set()
+
+    def process_get_item(self, key, result_queue):
+        result_queue.put(list(self.graph.__getitem__(key)))
+
+    def run(self):
```

Comment:
```suggestion
    def run(self):
        self.app.run()
        while True:
            req = self.read_write_queue.get()
            self.request_list.append(req)
```
This is where you would be reading from the queue and putting it into the list.